kustomize/install/bases/manager/manager.yaml

```diff
@@ -18,22 +18,24 @@ spec:
               fieldPath: metadata.namespace
         - name: CRUNCHY_DEBUG
           value: "true"
-        - name: RELATED_IMAGE_POSTGRES_14
-          value: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-14.1-0
-        - name: RELATED_IMAGE_POSTGRES_14_GIS_3.1
-          value: registry.developers.crunchydata.com/crunchydata/crunchy-postgres-gis:centos8-14.1-3.1-0
         - name: RELATED_IMAGE_POSTGRES_13
-          value: "registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-13.5-0"
+          value: "registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-13.5-1"
         - name: RELATED_IMAGE_POSTGRES_13_GIS_3.1
-          value: "registry.developers.crunchydata.com/crunchydata/crunchy-postgres-gis:centos8-13.5-3.1-0"
+          value: "registry.developers.crunchydata.com/crunchydata/crunchy-postgres-gis:centos8-13.5-3.1-1"
+        - name: RELATED_IMAGE_POSTGRES_14
+          value: "registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-14.1-1"
+        - name: RELATED_IMAGE_POSTGRES_14_GIS_3.1
+          value: "registry.developers.crunchydata.com/crunchydata/crunchy-postgres-gis:centos8-14.1-3.1-1"
```

Comment:
This is likely because we sync from the public repo, where PG 13 is still first in `manager.yaml`:

https://github.com/CrunchyData/postgres-operator/blob/0784a9390bce000447369438978ed4d4d5572ae7/config/manager/manager.yaml#L21-L28

So looks like we prob just need to get these in the same order across both repos.